v2/cmd/wails/internal/commands/common/usertags.go

```diff
@@ -0,0 +1,33 @@
+package common
+
+import (
+	"errors"
+	"strings"
+)
+
+// ParseUserTags parses the user tags from the command line
+func ParseUserTags(tags string) ([]string, error) {
+
+	if tags == "" {
+		return nil, nil
+	}
+
+	var userTags []string
+	separator := ""
+	if strings.Contains(tags, ",") {
+		separator = ","
+	}
+	if strings.Contains(tags, " ") {
+		if separator != "" {
+			return nil, errors.New("cannot use both space and comma separated values with `-tags` flag")
+		}
+		separator = ","
```

Comment:
Shouldn't this be a space?
```suggestion
		separator = " "
```